javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,94 @@
+//Item array removal :)
+
+const names = [
+    'Peter',
+    'Ahmad',
+    'Yana',
+    'kristina',
+    'Rasmus',
+    'Samuel',
+    'katrine',
+    'Tala',
+  ];
+  
+  names.splice(1, 1);
```

Comment:
This will remove the name at position 1 of the array, which happens to be "Ahmad" but what if "Ahmad" is in a different position. Try this.
```suggestion
const ahmadIndex = names.indexOf("Ahmad");
names = names.splice(ahmadIndex,1);
```